vision/google/cloud/vision/_http.py

```diff
@@ -52,6 +52,8 @@ def annotate(self, image, features):
         images = api_response.get('responses')
         if len(images) == 1:
             return Annotations.from_api_repr(images[0])
+        elif len(images) > 1:
+            raise ValueError('Multiple image processing is not yet supported.')
```

Comment:
`if len(images) == 0` then that means the service didn't detect anything it was asked too...most of the time.

There's actually something that I missed implementing originally. There's an `error` value in the response that I don't capture. That's in a PR to come shortly though.

I'll update to `NotImplementedError` real quick.